prefect_dbt/cloud/credentials.py

```diff
@@ -55,17 +61,123 @@ def trigger_dbt_cloud_job_run_flow():
     _block_type_name = "dbt Cloud Credentials"
     _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/5zE9lxfzBHjw3tnEup4wWL/9a001902ed43a84c6c96d23b24622e19/dbt-bit_tm.png?h=250"  # noqa
 
-    api_key: SecretStr
-    account_id: int
-    domain: str = "cloud.getdbt.com"
-
-    def get_administrative_client(self):
+    api_key: SecretStr = Field(
+        default=...,
+        title="API Key",
+        description="A dbt Cloud API key to use for authentication.",
+    )
+    account_id: int = Field(
+        default=..., title="Account ID", description="The ID of your dbt Cloud account."
+    )
+    domain: str = Field(
+        default="cloud.getdbt.com",
+        description="The base domain of your dbt Cloud instance.",
+    )
+
+    def get_administrative_client(self) -> DbtCloudAdministrativeClient:
         """
         Returns a newly instantiated client for working with the dbt Cloud
         administrative API.
+
+        Returns:
+            An authenticated dbt Cloud administrative API client.
         """
         return DbtCloudAdministrativeClient(
             api_key=self.api_key.get_secret_value(),
             account_id=self.account_id,
             domain=self.domain,
         )
+
+    def get_metadata_client(self) -> DbtCloudMetadataClient:
+        """
+        Returns a newly instantiated client for working with the dbt Cloud
+        metadata API.
+
+        Example:
+            Sending queries via the returned metadata client:
+            ```python
+            from prefect_dbt import DbtCloudCredentials
+
+            credentials_block = DbtCloudCredentials.load("test-account")
+            metadata_client = credentials_block.get_metadata_client()
+            query = \"\"\"
```

Comment:
Is the `\` needed here?

Reply:
Yeah, need to escape the quotes so that the docstring continues.